test/causal_crdt_test.exs

```diff
@@ -169,4 +170,48 @@ defmodule CausalCrdtTest do
     refute Map.has_key?(DeltaCrdt.read(c1), "key")
     refute Map.has_key?(DeltaCrdt.read(c2), "key")
   end
+
+  @tag :slow
+  @tag timeout: 600_000
+  test "adding a lot of entries" do
+    {:ok, c1} = DeltaCrdt.start_link(AWLWWMap, sync_interval: 500)
+    {:ok, c2} = DeltaCrdt.start_link(AWLWWMap, sync_interval: 500)
+    DeltaCrdt.set_neighbours(c1, [c2])
+    DeltaCrdt.set_neighbours(c2, [c1])
+
+    for key <- 1..100_000, do: DeltaCrdt.mutate(c1, :add, [key, %{}], 60_000)
+
+    Process.sleep(60_000)
+
+    assert map_size(DeltaCrdt.read(c1)) == 100_000
+    assert map_size(DeltaCrdt.read(c2)) == 100_000
+  end
+
+  test "adding a neighbour with own entries to existing pair" do
+    {:ok, c1} = DeltaCrdt.start_link(AWLWWMap, sync_interval: 20)
+    {:ok, c2} = DeltaCrdt.start_link(AWLWWMap, sync_interval: 20)
+    DeltaCrdt.set_neighbours(c1, [c2])
+    DeltaCrdt.set_neighbours(c2, [c1])
+
+    DeltaCrdt.mutate(c1, :add, ["step1", 1])
+
+    Process.sleep(50)
+
+    # Create a new neighbour that already has data:
+    {:ok, c3} = DeltaCrdt.start_link(AWLWWMap, sync_interval: 20)
+    DeltaCrdt.mutate(c3, :add, ["step2", 2])
+
+    Process.sleep(50)
+
+    # Connect them
+    DeltaCrdt.set_neighbours(c3, [c1])
+    DeltaCrdt.set_neighbours(c1, [c3])
```

Comment:
This line disconnects c1 from c2, resulting in c2 only sending updates to c1, and receiving them from no node.

Test works with `DeltaCrdt.set_neighbours(c1, [c2, c3])`